test_files/expected/tester_config.yaml

```diff
@@ -18,4 +18,6 @@ nillable:
     anticipated: anticipated_alt
 
 label-contents:
-  version_id: 1.1
\ No newline at end of file
+  version_id: 1.1
+  File:
```

Comment:
Aha! I guess I see where the `File` variable comes from in my earlier question about the template, but this isn't in the user documentation. Also I don't know what "file" it's talking about. Maybe it needs a more explicit name?

Reply:
I have changed the template to instead reference File_Area_(Ancillary/Metadata) to be more specific.